.github/workflows/course-schedule.yml

```
@@ -0,0 +1,25 @@
+name: "Course Schedule Updates"
+on:
+  - pull_request
```

Perhaps this will be nice to avoid triggering it on other files:

```suggestion
on:
  pull_request:
    paths:
    - 'src/**.md'
```

Down the road, I could also imagine changing the logic to not add a comment unless the time changed. So if someone fixes a typo, they probably don't need the timing information.